pages/linux/opkg.md

```diff
@@ -0,0 +1,31 @@
+# opkg
+
+> A lightweight package manager used to install OpenWrt packages.
+
+- Install a package:
+
+`opkg install {{wget}}`
```

Comment:
Just a minor query, would it be better to use `{{package}}` instead of `{{wget}}` for each example. I understand that `wget` is an example, but it would just add consistency with the `{{package(s)}}` usage.